module2/src/com/epam/onedimensionalarrays/RealNumbersArrayAnalyser.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.onedimensionalarrays;
+
+
+public class RealNumbersArrayAnalyser {
+
+    public static int[] characterizeArray(int[] array, int negativeElements, int positiveElements, int zeroElements) {
```

Comment:
just use here local variables:
```suggestion
    public static int[] characterizeArray(int[] array) {
    int negativeElements, positiveElements, zeroElements = 0;
```